### DISCUSSION TITLE
Show account name in iOS widgets

### DISCUSSION THREAD

**Discussion by SVilgelm** [community/community#77058] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.139.1 (116337430)
Device: iPad14,3 (iOS 17.1.1)
</details>

In case of using the multiple accounts and the desktop widgets, it is not clear for what account a widget shows the data. It would be a great to show an account name in widget's title

**Comment by HenSquared** [comment]
Nice idea :) 

I've created an internal issue to track this and discuss with our team. Thanks for the feedback!

**Reply by HenSquared** [reply]
@SVilgelm , is there a specific widget you'd like to include username info? (Pull Requests or contribution graph)

For now, you can long press on the widget and select 'account' to choose a specific account to use (or just to check which account is displaying)

**Reply by SVilgelm** [reply]
I know about how to choose the account. Just if I have several accounts and want to see the assigned pull request for each account, I need to use a widget per account, but then it is hard to recognize which widget is for witch account at first glance.
